MiniApp/Classes/core/RealMiniApp.swift

```diff
@@ -320,6 +319,15 @@ internal class RealMiniApp {
     func getCachedManifestData(appId: String) -> MiniAppManifest? {
         return self.miniAppManifestStorage.getManifestInfo(forMiniApp: appId)?.miniAppManifest
     }
+
+    func isDeviceOfflineError(error: NSError) -> Bool {
+        if error.domain == MASDKErrorDomain {
+            return (error as? MASDKError)?.isDeviceOfflineDownloadError() ?? false
+        } else if offlineErrorCodeList.contains(error.code) {
+            return true
+        }
+        return false
```

Comment:
sorry I did not notice, but this can also be simplified as
```swift
        if error.domain == MASDKErrorDomain {
            return (error as? MASDKError)?.isDeviceOfflineDownloadError() ?? false
        } 

        return offlineErrorCodeList.contains(error.code)
```